src/components/AppBar/NavBar.tsx

```diff
@@ -77,22 +80,27 @@ const NavBar: React.FC = () => {
         )}
         <Link
           to="/"
-          style={{ display: 'inline-flex' }}
+          style={{ display: 'inline-flex', zIndex: 10000 }}
```

Comment:
manually moving the logo to be on top of the absolutely-positioned div that wraps the search bar so the logo is clickable/not covered